src/main/java/com/github/tomakehurst/wiremock/matching/RequestPattern.java

```diff
@@ -128,6 +131,73 @@ public String getName() {
         };
   }
 
+  public RequestPattern(
```

Comment:
This is when we switch to the `withSomething()` pattern 💀 
But I think we could at least avoid code duplication in the constructor, right?

Reply:
Now RequestPattern with RequestMethod method use RequestPattern with Set<RequestMethod> methods